packages/react/src/components/CardEditor/CardEditForm/CardEditForm.jsx

```diff
@@ -117,10 +117,10 @@ const defaultProps = {
   isSummaryDashboard: false,
   dataSeriesItemLinks: null,
   actions: {
-    onEditDataItem: null,
+    onEditDataItem: () => {},
```

Comment:
add `/* istanbul ignore next */` above the default props declaration on line 83 to ignore these no-ops. Do the same for each file where no ops were added